Assets/FrostedGlass/Scripts/CommandBufferBlur.cs

```diff
@@ -13,8 +13,8 @@ public class CommandBufferBlur : MonoBehaviour
     Camera _Camera = null;
     CommandBuffer _CommandBuffer = null;
 
-	private int prevW = 0;
-	private int prevH = 0;
+    private int prevW = 0;
```

Comment:
Please, use the file naming convention, something like Vector2 _ScreenResolution = Vector2.zero